pyproject.toml

```diff
@@ -0,0 +1,21 @@
+[tool.poetry]
+name = "aldjemy"
+version = "1.0.0"
+description = "Integration SQLAlchemy in Django projects for effective query building purposes"
+authors = ["Mikhail Krivushin"]
+license = "BSD 3-Clause \"New\" or \"Revised\" License"
+
+[tool.poetry.dependencies]
+python = "^3.6"
+SQLAlchemy = "^1.3.20"
+
+[tool.poetry.dev-dependencies]
+tox = "^3.20.1"
```

Comment:
I would not recommend having `tox` in the dev-dependencies. I view tox as a higher-level command that uses poetry, not the other way around.

I'd be on board with dropping `tox` in our build matrix if that seems better. I'm not sure what the ideal way to do this integration is, so I'd be fine with leaving off `tox` while we figure out how to use GitHub actions and poetry well.